pkg/agent/proxy/proxier.go

```diff
@@ -81,6 +89,7 @@ func (p *proxier) removeStaleServices() {
 			continue
 		}
 		svcInfo := svcPort.(*types.ServiceInfo)
+		klog.Infof("Removing stale Service: %s %s", svcPortName.Name, svcInfo.String())
```

Comment:
maybe should be `V(2)` or even higher?

same for the other new Info log messages below

Reply:
Thanks for mentioning, updated.